jwst/ramp_fitting/tests/test_ramp_fit_step.py

```diff
@@ -1,9 +1,21 @@
 import pytest
+import logging
 import numpy as np
 
 from stdatamodels.jwst.datamodels import dqflags, RampModel, GainModel, ReadnoiseModel
 
-from jwst.ramp_fitting.ramp_fit_step import RampFitStep
+from jwst.ramp_fitting.ramp_fit_step import RampFitStep, set_groupdq
+from jwst.tests.helpers import LogWatcher
+
+@pytest.fixture
+def log_watcher(monkeypatch):
+    # Set a log watcher to check for a log message at any level
+    # in RampFitStep
+    watcher = LogWatcher('')
+    logger = logging.getLogger('stpipe')
```

Comment:
I think it needs to be this:

```suggestion
    logger = logging.getLogger('jwst.ramp_fitting.ramp_fit_step')
```

because the message comes from a function defined in `ramp_fit_step`, which is using a locally defined log, from the top of the file: `log = logging.getLogger(__name__)`.  Here, `__name__` is `jwst.ramp_fitting.ramp_fit_step`.

Reply:
This worked!  Not sure what I did wrong when I tried it yesterday.